src/Private/Context.ps1

```diff
@@ -936,6 +936,13 @@ function Set-PodeWebConfiguration {
         Compression      = @{
             Enabled = [bool]$Configuration.Compression.Enable
         }
+        OpenApi          = @{
+            DefaultDefinitionTag = [string](Protect-PodeValue -Value $Configuration.OpenApi.DefaultDefinitionTag -Default 'default')
+        }
+    }
+
+    if ($Configuration.OpenApi -and $Configuration.OpenApi.ContainsKey('UsePodeYamlInternal')) {
+        $Context.Server.Web.OpenApi.UsePodeYamlInternal = $Configuration.OpenApi.UsePodeYamlInternal
     }
```

Comment:
This block is almost identical to the block just beneath it 😄

Reply:
it was wrong I was sure I removed it. But I was wrong